### DISCUSSION TITLE
어떤 게임이 갓겜인가

### DISCUSSION THREAD

**Discussion by Dangtae** [happyCoVid-19/PS#2] [General]
온라인, pc, 콘솔 통틀어 가장 갓겜은 무엇인가

**Comment by minrongtic** [comment]
난 옛날에 야후 꾸러기에 있었던 아르피아

**Reply by Dangtae** [reply]
아르피아 갓겜 인정

**Comment by limhongcheol** [comment]
1.Maplestory
2.League of Legends
3.Fifa Online 4
4.Europa Universalies 4
5.Football Manager 2023
6.Hearth Stone

**Reply by minrongtic** [reply]
Can you speak Korean?

**Reply by limhongcheol** [reply]
Sorry, 한쿡말 조금 어려워!

**Comment by Dangtae** [comment]
갠적으로 기억에 남는 건
디트로이트 비컴 휴먼

바이오하자드 RE: 2도 상당히 재밌음

**Comment by Solsol1014** [comment]
장서현 인생겜 
1. 문명
2. 팩토리오
